main.go

```
@@ -0,0 +1,24 @@
+
+func main() {
+    nameRepo := repository.NewNameRepository()
+    nameHandler := handler.NewNameHandler(nameRepo)
+
+    application := app.NewApp()
+    fmt.Println("app start!", application)
+
+    application.Router.Mount("/names", route.NameRouter(nameHandler))
+    fmt.Println("route set up done!!")
+
+    http.ListenAndServe(":3000", application.Router)
```

golangci-lint (1.64.8)

Error return value of `http.ListenAndServe` is not checked

(errcheck)